openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/TestSuiteIngestion.tsx

```diff
@@ -188,15 +208,16 @@ const TestSuiteIngestion: React.FC<TestSuiteIngestionProps> = ({
       <Col span={24}>
         {isIngestionCreated ? (
           <SuccessScreen
-            isAirflowSetup
             handleDeployClick={handleDeployClick}
             handleViewServiceClick={handleViewTestSuiteClick}
+            isAirflowSetup={isAirflowRunning}
             name={`${testSuite?.name}_${PipelineType.TestSuite}`}
             showDeployButton={showDeployButton}
             showIngestionButton={false}
             state={FormSubmitType.ADD}
             successMessage={getSuccessMessage}
             viewServiceText="View Test Suite"
+            onCheckAirflowStatus={handleAirflowStatusCheck}
```

Comment:
Do we need this prop as we are passing the `isAirflowSetup={isAirflowRunning}` prop?

Reply:
yes we need these props, we have one button to recheck the status